apps/playground/src/app/quiz/components/question/question.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, OnChanges, SimpleChanges, Input, Output, EventEmitter } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { QuizQuestion } from '../../model/QuizQuestion';
+
+@Component({
+  selector: 'codelab-quiz-question',
+  templateUrl: './question.component.html',
+  styleUrls: ['./question.component.scss']
+})
+export class QuestionComponent implements OnInit, OnChanges {
+  @Output() answer = new EventEmitter<string>();
+  @Output() formGroup: FormGroup;
+  @Input() question: QuizQuestion;
+  option = '';
+  grayBorder = '2px solid #979797';
+
+  constructor() {}
+
+  ngOnInit() {
+    this.buildForm();
+  }
+
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.question && changes.question.currentValue && !changes.question.firstChange) {
+      this.formGroup.patchValue({answer: ''});
+    }
+  }
+
+  buildForm() {
+    this.formGroup = new FormGroup({
+      answer: new FormControl(['', Validators.required])
+    });
+  }
+
+  radioChange(answer: string) {
+    this.question.selectedOption = answer;
+    this.answer.emit(answer);
+    this.displayExplanation();
+  }
+
+  displayExplanation(): void {
+    const questionElem = document.getElementById('question');
+    if (questionElem !== null) {
+      questionElem.innerHTML = 'Option ' + this.question.answer + ' was correct because ' + this.question.explanation + '.';

```

Comment:
Please move this to the template

Reply:
added explanation to dependency-injection-quiz template and added scss to ts file